README.md

```diff
@@ -6,17 +6,21 @@
 [go-report-image]: https://goreportcard.com/badge/github.com/temporalio/temporal
 [go-report-url]: https://goreportcard.com/report/github.com/temporalio/temporal
 
-# Temporal  
+# Temporal
 
-Temporal is a microservice orchestration platform which enables developers to build scalable applications without sacrificing productivity or reliability.
-Temporal server executes units of application logic, Workflows, in a resilient manner that automatically handles intermittent failures, and retries failed operations.
+Temporal is a durable execution platform that enables developers to build scalable applications without sacrificing productivity or reliability.
```

Comment:
Finally! It is :man_facepalming: that we are still "microservice orchestration platform" here.